content/docs/actors/publishing/seo_and_promotion.md

```diff
@@ -77,7 +77,7 @@ The description is used by search engines to understand what the actor is about,
 <br/>
 > **AVOID**: This crawler will search through results in Amazon and offers you...
 
-If you’re not sure about similar phrases, check out the Google suggest tool in the search and it will show you some hints and tips. Of course, the description should primarily be about what the actor does, but it’s always a good idea to enrich the text with popular keywords.
+If you’re not sure about similar phrases, check out the Google suggest tool in the search, and it will show you some hints and tips. Of course, the description should primarily be about what the actor does, but it’s always a good idea to enrich the text with popular keywords.
```

Comment:
```suggestion
If you’re not sure about similar phrases, check out the Google suggest tool in the search and it will show you some hints and tips. Of course, the description should primarily be about what the actor does, but it’s always a good idea to enrich the text with popular keywords.
```